Cargo.toml

```diff
@@ -11,7 +11,7 @@ documentation = "https://docs.rs/crossfire"
 keywords = ["async", "non-blocking", "lock-free", "channel"]
 categories = ["concurrency", "data-structures"]
 exclude = ["/ci/*", "/bors.toml"]
-description = "channels for async and threads"
```

Comment:
What's problem in that description?

Reply:
sorry about that, i think when i pasted the cargo version it replaced that line on accident. added back.